pages/index.html

```diff
@@ -127,10 +127,25 @@ <h2 class="bannerbox__title">I need a ride</h2>
                                 <input type="checkbox" name="TwoWayTripNeeded" id="rideReturn" /> I need a two-way trip.
                             </label>
                         </div>
+                        <div class="form-group checkbox">
+                            <label for="riderDoesNotSpeakEnglish">
+                                <input type="checkbox" name="TwoWayTripNeeded" id="riderDoesNotSpeakEnglish" /> I can only communicate in a language other than English.
+                            </label>
+                        </div>
                         <div class="form-group">
-                            <label for="RiderAccommodationNotes">Notes <i class="optional">Optional</i></label>
+                            <label for="RiderAccommodationNotes">Other Requirements <i class="optional">Optional</i></label>
                             <textarea class="form-input" id="RiderAccommodationNotes"
                             placeholder="Accommodation requirements e.g. service animal, assistance folding equipment, assistance entering/exiting the vehicle, child car seat or booster (please include age), must also collect non-voting assistant or companion, or anything else the driver should know." name="RiderAccommodationNotes" rows="4"></textarea>
+                            <small>
+                                Please let us know of any other accommodation requirements.
+                                <ul>
```

Comment:
Placing `<ul>` and `<li>` elements inside a `<small>` is invalid HTML :(
I recommend replacing the `<small>` with a `<div>` (?) with either a new `small-text` class or an inline style, with `font-size: 80%`.
Does that sound like a decent solution to you?

Although maybe you could just apply the small tag to the first line, and add the class to the `<ul>`, like this: 
```
<small>Please let us know of any other accommodation requirements.</small>
<ul class="small-text">
  <li>Service animal</li>
  <li>Assistance folding equipment</li>
  <li>Assistance entering/ exiting the Vehicle</li>
  <li>Child car seat or booster (please include age)</li>
  <li>Etc.</li>
</ul>
```
```
.small-text {
   font-size: 80%;
}
```